cmd/kind/load/docker-image/docker-image.go

```diff
@@ -68,6 +72,12 @@ func NewCommand() *cobra.Command {
 }
 
 func runE(flags *flagpole, cmd *cobra.Command, args []string) error {
+	imageName := args[0]
+	// Check that the image exists locally, if not return error
+	cmdDocker := exec.Command("docker", "inspect", "--type=image", imageName)
```

Comment:
This logic should be in the docker package

Reply:
done